graphics/lvgl/README.md

```diff
@@ -0,0 +1,21 @@
+# Usage
+
+Import with `#include <lvgl/lvgl.h>`.
+
+Upstream example is present at `examples/lvgldemo`.
+
+LVGL can be used with framebuffer device. To find example boards with this
+preconfigured, search for `CONFIG_GRAPHICS_LVGL=y` in `defconfig` files.
+All of them have also `CONFIG_VIDEO_FB=y` present.
+
+As a second option, LVGL can talk to a display driver and explicitly draw line by line.
+For this case, there is no preconfigured board present. Go to _Porting_ section
+of upstream documentation for more hints.
```

Comment:
Not sure the second approach is possible.

Reply:
Yes, I am using it. Although I bypassed NuttX display driver because it was considerably slower at that time (NuttX 7.24). Or I didn't know how to use it better.

NuttX framebuffer as far as I understand needs width x height x depth amount of memory. It means that it requires external RAM. But LVGL don't have such limitation. It can draw almost directly with some few kilobyte buffer.

Reply:
In my own use of littevgl in NuttX I actually draw using a custom LCD character driver to avoid the overhead of using the framebuffer->LCD emulation layer of NuttX (you can see the driver here: https://gitlab.com/nuttx_projects/lcd_dev). 
Just FYI.

Reply:
So there is some emulation layer that can be enabled? Maybe we should add this information and your custom solution to the README to help people who want to use it.

Reply:
No, this is not standard, I use it on my own code. I could propose it in a pull-request. I never did since I extended the LCD interface to accept drawing multiple rows at once (also due to overhead of individual putrun()), but I could leave that out. I would not add something to the README if it is not part of NuttX codebase itself.

Reply:
The topic of drawing multiple-lines comes back from time to time. I would also be in favor of having such interface beacuse it seems to be a basic use case. Somebody was trying to add it few months ago, but I don't know where it ended up.

I understand, these READMEs seems to be more serious. For me README is just a way to communicate. If someone needs to know something and there are some solutions, README would be the place.

Anyway I think it is enough for this PR then.

Reply:
@v01d why not enhance the driver layer? @w8jcik also look for a solution which mean this is a common requirement.

Reply:
@xiaoxiang781216 I can open a PR for my LCD character driver. As per the ability to send multiple rows, I propose that on the mailing list and did not get much response. We can discuss it further again.

Reply:
At least, @w8jcik is also interesting your solution. We can talk about the detail in your PR.